assets/script.js

```diff
@@ -25,7 +25,7 @@ var weatherIconEl = document.querySelector("#weather-icon")
 // Invalid message
 var invalidInputEl = document.querySelector('#invalid-input')
 
-var weatherIconEl = document.querySelector("#weather-icon")
```

Comment:
this var was duplicated and i needed to add one for the temperature icon so i just edited it